pynuodb/session.py

```diff
@@ -429,15 +429,17 @@ def __readFully(self, msgLength, timeout=None):
 
     def close(self, force=False):
         """ Close the current socket connection with the server """
-        if not self.__sock:
+        if self.__sock is None:
             return
 
         try:
             if force:
-                self.__sock.shutdown(socket.SHUT_RDWR)
-
-            if self.__sock:
-                self.__sock.close()
+                try:
+                    self.__sock.shutdown(socket.SHUT_RDWR)
```

Comment:
I actually don't have any idea what this "force" parameter is supposed to do or why it's here: it's been here since the initial commit of this code.